developer/src/kmc-kmn/src/kmw-compiler/messages.ts

```diff
@@ -39,6 +39,10 @@ export class KmwCompilerMessages extends KmnCompilerMessages {
     `Statement ${o.code} is not currently supported in context`);
   static Error_NotSupportedInKeymanWebOutput = (o:{code:string}) => m(this.ERROR_NotSupportedInKeymanWebOutput,
     `Statement ${o.code} is not currently supported in output`);
+  static Warn_HelpFileMissing = (o:{filename: string, e:any}) => m(this.WARN_HelpFileMissing,
+    `File ${o.filename} could not be loaded: ${(o.e??'').toString()}`);
```

Comment:
`${(o.e??'')` - interesting syntax!!

Reply:
Yes, [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) in JS is very useful